internal/test/test.go

```diff
@@ -0,0 +1,40 @@
+package test
```

Comment:
i didnt want to name it testing because every test already imports the builtin testing package. so you always have to alias it to testing2 or ltesting or something

im open to a better name